JosephYeh.md

```diff
@@ -1,16 +1,17 @@
 ### Last week
 
-* [Marketplace]
-  - [FIXED] Bug 1263807 - In Web Apps, navigation between first row and second row will cause screen scrolling
-
 * [Firefox]
-  - Practice MozReview (Bug 1259355)
+  - [FIXED] Bug 530999 - Login manager UI should show site favicons
 
 ### This Week
 
 * [Marketplace]
   - [NEW] Bug 1253240 - [TV] Adding "Category" feature for TV Web Apps Discovery Portal
+  - [NEW] Bug 1265664 - [TV][Marketplace] Move card description text to outside the text.
   - Brown Bag Marketplace slides
 
 * [Firefox]
-  - [WIP] Bug 530999 - Login manager UI should show site favicons
+  - No progress.
+
+* [Academy NCU 2016]
+  - Preparing slides and exercise.
```

Comment:
You should let me know you are in the project earlier before committing anything.